### ISSUE TITLE
Value of onConflict must be one of OnConflictStrategy

### ISSUE THREAD

**Issue by AbiMulLyA** [Amir-P/flat#25]
Hi.. thanks for forking the floor plugin.. could u fix this one?.. this also issue on floor when I run build_runner command
This is related issue on floor : https://github.com/vitusortner/floor/issues/641

![image](https://user-images.githubusercontent.com/42445812/171586980-8f96acf2-29f1-4fb6-8071-6da321d7ee71.png)

**Comment by Amir-P** [comment]
It should be fixed in the latest release. @AbiMulLyA 

```yaml
dependencies:
  flat_orm: ^1.5.1+1

dev_dependencies:
  flat_generator: ^1.6.1+1
```

**Comment by AbiMulLyA** [comment]
I'm using the latest version with analyzer 4.1.0.. but still got the error @Amir-P

**Comment by Amir-P** [comment]
With latest release I think it should be fixed (see #19 and #20). Can you share your code so I can reproduce the error? @AbiMulLyA

**Comment by AbiMulLyA** [comment]
After I do `flutter pub cache repair`, it's solved.. Thank you !
